telluric/georaster.py

```diff
@@ -125,19 +125,20 @@ def _prepare_other_raster(one, other):
     return other
 
 
-def _merge(one, other, merge_strategy=MergeStrategy.UNION):
+def _merge(one, other, merge_strategy=MergeStrategy.UNION, common_bands=None):
```

Comment:
can you explain why you had to add the band_names argument ?

Reply:
I am using it in the line 194, 
Before the change, we did it twice, in line 194 and right afterward because of the recursion also [here](https://github.com/satellogic/telluric/pull/41/files#diff-6dcac23673708582303bd00fc00849e4L140).

Maybe I should change the name to requested_bands